src/delphi/__init__.py

```diff
@@ -1,3 +1,5 @@
 from beartype.claw import beartype_this_package  # <-- hype comes
 
+from . import eval
+
```

Comment:
why?

Reply:
I still require this so my I can import everything. For WHATEVER reason an empty __init__.py file will not allow me to import it. (of course, I used `pip install -e .`)